### ISSUE TITLE
is there any way that alow prender middleware return pages of specific routes 

### ISSUE THREAD

**Issue by VienNguyen113** [prerender/prerender-node#172]

**Comment by thoop** [comment]
Could you give an example? You can use the whitelist and blacklist as seen on the readme to only allow specific routes to be sent to Prerender.

**Comment by VienNguyen113** [comment]
@thoop yeah! i'used spliting code in my app! it's mean the Js file that represent for a route just downloaded to excute if only the user vistit that route! example: Home.js file will be downloaded if user visit my-app/home, the problem is come from here, i saw the middleware just return the js file of the master route.
 <Route path="/" component={MasterPage}/>
<Route path="/home" component={HomePage}/>

**Comment by thoop** [comment]
I'm not sure what you mean. If a crawler accesses the root "/" URL then Prerender would render the root "/" URL. If a crawler accessed the "/home" URL then Prerender would render the "/home" URL.

Are you seeing something else happening?

**Comment by thoop** [comment]
As long as the prerender-node middleware is running before you serve back your index.html page, then everything should work properly. Do you have an upstream server like nginx/apache that is modifying the URL to just the "/" route before sending it to your node server?

**Comment by VienNguyen113** [comment]
oh! i did'nt use a webserver yet! i just set-up a pre-render server and an express server with the middleware

**Comment by thoop** [comment]
That should be fine then. Can you post your express configuration with any private information removed?
